docs/src/Text.doc.js

```diff
@@ -21,11 +21,13 @@ card(
         name: 'align',
         type: `"left" | "right" | "center" | "justify"`,
         defaultValue: 'left',
+        id: 'align',
       },
       {
         name: 'bold',
         type: 'boolean',
         defaultValue: false,
+        id: 'styles',
```

Comment:
since each prop name has to be unique on the component, maybe we should just reuse the name field for the id as well all around - this `bold` to `styles` naming is a bit confusing at first glance

Reply:
@pfarejowicz I elected a separate prop for two reasons.
1st, it made it easier to only wrap prop that led to examples in a `<Link/>`
2nd, in the case of the `<Text/>` docs both bold and italic lead to the same example so using name didnt seem quite right.  Watcha think?